lib/sinon/stub.js

```diff
@@ -18,6 +18,14 @@ function stub(object, property) {
         throw new TypeError("stub(obj, 'meth', fn) has been removed, see documentation");
     }
 
+    if (
+        object &&
+        typeof Symbol !== "undefined" &&
+        object[Symbol.toStringTag] === "Module"
+    ) {
+        throw new TypeError("No support for stubbing ES Modules");
```

Comment:
This sound a bit like "Sinon does not have this feature", but really, it's impossible. Maybe "ES modules cannot be stubbed"?